<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background: #f5f5f5;
	font-family: 'Montserrat';
}

QPushButton {
	font-size: 12px; font-weight: bold;
	text-transform: uppercase;
	border: 2px solid #263238;
	padding: 10px;
}

QPushButton:hover {
	background: rgba(194,24,91,0.2);
	
}
QPushButton:pressed {
	background: #C2185B;
}

QLineEdit {
	font-size: 11px;
	border-bottom: 1px solid #263238;
	padding: 10px;
	border-style: outset;
}

QLabel {
	font-size: 14px;
}

QSpinBox {
	font-size: 14px;
}

QLineEdit:focus {
	border-bottom: 2px solid #1A237E;
}

QTextEdit {
	font-size: 11px;
	border: 1px solid #263238;
	padding: 10px;
	border-style: outset;
}
QTextEdit:focus {
	border: 2px solid #1A237E;
}


QTextEdit#txtOutput {
	background: #455a64; color: #fafafa;
}

QPushButton#btnOptimize, QPushButton#btnMakeGif {
	border-color: #66AC00;
}
QPushButton#btnOptimize:hover, QPushButton#btnMakeGif:hover {
	background-color: #AFFC41;
}
QPushButton#btnListImagesRemove:hover {
	background-color: #FC4169;
	border-color: #FC4169;
}

QLabel#labelGeneralOptions, QLabel#labelGifOptions {
	font-weight: bold;
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="3" column="0" alignment="Qt::AlignLeft">
       <widget class="QFrame" name="hFrameOption3">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnOptimize">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Optimize</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMakeGif">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Make GIF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnBrowseFolder">
           <property name="text">
            <string>Image Folder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnBrowseImageFolder">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="formImageFolder">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="hFrameOption1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
           <property name="textVisible">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="textDirection">
            <enum>QProgressBar::TopToBottom</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0" alignment="Qt::AlignLeft">
       <widget class="QFrame" name="hFrameAction">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="hLayoutOptions_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="hFrameOption2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <item row="2" column="0">
            <widget class="QLabel" name="labelWidth">
             <property name="text">
              <string>Resize width</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="labelSliderQuality">
             <property name="text">
              <string>Quality</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QComboBox" name="comboBoxFormat">
             <property name="insertPolicy">
              <enum>QComboBox::NoInsert</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QComboBox::AdjustToContents</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelGeneralOptions">
             <property name="text">
              <string>General options</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="chkReplaceSource">
             <property name="text">
              <string>Replace original</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="chkFileMode">
             <property name="text">
              <string>File mode</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QSlider" name="hSliderQuality">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>5</number>
             </property>
             <property name="pageStep">
              <number>5</number>
             </property>
             <property name="value">
              <number>80</number>
             </property>
             <property name="sliderPosition">
              <number>80</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
             <property name="invertedControls">
              <bool>false</bool>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="spinBoxBasewidth">
             <property name="suffix">
              <string> px</string>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>1980</number>
             </property>
             <property name="singleStep">
              <number>128</number>
             </property>
             <property name="stepType">
              <enum>QAbstractSpinBox::DefaultStepType</enum>
             </property>
             <property name="value">
              <number>1280</number>
             </property>
             <property name="displayIntegerBase">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Ext</string>
             </property>
            </widget>
           </item>
           <item row="2" column="6">
            <widget class="QLabel" name="labelSliderQualityValue">
             <property name="text">
              <string>80</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <widget class="QListWidget" name="listWidgetImages">
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="tabKeyNavigation">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DragDrop</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::MoveAction</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="movement">
         <enum>QListView::Free</enum>
        </property>
        <property name="flow">
         <enum>QListView::TopToBottom</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QFrame" name="horizontalFrame">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QFrame" name="makeGifFrame">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item row="2" column="2">
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QLabel" name="labelMakeGifDuration">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Duration</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBoxMakeGifDuration">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimum">
                 <number>10</number>
                </property>
                <property name="maximum">
                 <number>60000</number>
                </property>
                <property name="singleStep">
                 <number>50</number>
                </property>
                <property name="value">
                 <number>1000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="3">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QLabel" name="labelMakeGifRepeat">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Repeat</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBoxMakeGifRepeat">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximum">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="2" colspan="5">
             <widget class="QLabel" name="labelGifOptions">
              <property name="text">
               <string>GIF Maker options</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
